docs/src/main/paradox/docs/delta/api/current/views/composite-view-api.md

```diff
@@ -249,259 +256,354 @@ The following example creates a Composite view containing 3 sources and 2 projec
+:   @@snip [create-put.sh](../assets/views/composite/create-put.sh)
 
 Payload
-:   @@snip [composite-view-es-payload.json](../assets/views/composite-view-es-search-payload.json)
+:   @@snip [payload.json](../assets/views/composite/payload.json)
 
 Response
-:   @@snip [composite-view-es-search.json](../assets/views/composite-view-es-search.json)
+:   @@snip [created-put.json](../assets/views/composite/created-put.json)
+
+## Update
+
+This operation overrides the payload.
 
-### Search Documents in all projections
+In order to ensure a client does not perform any changes to a resource without having had seen the previous revision of
+the view, the last revision needs to be passed as a query parameter.
 
 ```
-POST /v1/views/{org_label}/{project_label}/{view_id}/projections/_/_search
+PUT /v1/views/{org_label}/{project_label}/{view_id}?rev={previous_rev}
   {...}
 ```
 
-The supported payload is defined on the 
-@link:[ElasticSearch documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-body.html){ open=new }
+... where `{previous_rev}` is the last known revision number for the view.
+
+@@@ note { .warning }
 
+Updating a view creates a new Elasticsearch index and deletes the existing one. The indexing process will start from the
+beginning.
+
+@@@
 
 **Example**
 
 Request
-:   @@snip [composite-view-es-search.sh](../assets/views/composite-view-es-search.sh)
+:   @@snip [update.sh](../assets/views/composite/update.sh)
 
 Payload
-:   @@snip [composite-view-es-payload.json](../assets/views/composite-view-es-search-payload.json)
+:   @@snip [payload.json](../assets/views/composite/payload.json)
 
 Response
-:   @@snip [composite-view-es-search.json](../assets/views/composite-view-es-search.json)
+:   @@snip [updated.json](../assets/views/composite/updated.json)
+
+## Tag
 
+Links a view revision to a specific name.
 
-### SPARQL query in a projection
+Tagging a view is considered to be an update as well.
 
 ```
-POST /v1/views/{org_label}/{project_label}/{view_id}/projections/{projection_id}/sparql
-  {query}
+POST /v1/views/{org_label}/{project_label}/{view_id}/tags?rev={previous_rev}
+  {
+    "tag": "{name}",
+    "rev": {rev}
+  }
 ```
 
+... where
+
+- `{previous_rev}`: is the last known revision number for the resource.
+- `{name}`: String - label given to the view at specific revision.
+- `{rev}`: Number - the revision to link the provided `{name}`.
+
+**Example**
+
+Request
+:   @@snip [tag.sh](../assets/views/composite/tag.sh)
+
+Payload
+:   @@snip [tag.json](../assets/tag.json)
+
+Response
+:   @@snip [tagged.json](../assets/views/composite/tagged.json)
+
+## Deprecate
+
+Locks the view, so no further operations can be performed. It also stops indexing any more resources into it and deletes the underlying index.
+
+Deprecating a view is considered to be an update as well.
+
+@@@ note { .warning }
+
+Deprecating a view deletes the view index, making the view not searchable.
```

Comment:
Same here:  deletes all the underlying Elasticsearch indices and Blazegraph namespaces.